specification/app/resource-manager/Microsoft.App/stable/2024-03-01/Diagnostics.json

```diff
@@ -432,6 +432,154 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/jobs/{jobName}/detectors": {
+      "get": {
+        "tags": [
+          "Jobs",
+          "Diagnostics"
+        ],
+        "summary": "Get the list of diagnostics for a given Container App Job.",
```

Comment:
I think we should be consistent with diagnostics or detectors usage. It is not clear both are same